dadguide/models/monster_stats.py

```diff
@@ -1,4 +1,5 @@
 from typing import Literal
+from dadguide.token_mappings import AWAKENING_TOKENS, AWOKEN_SKILL_MAP, PLUS_AWOKENSKILL_MAP, AwokenSkills
```

Comment:
I believe the only one getting used is AwokenSkills and the others are unnecessary